src/components/pages/ManageDomain/ManageDomain.tsx

```diff
@@ -20,18 +101,17 @@ function ManageDomain() {
   const [, dispatchModalState] = useModalState();
   const { data: primaryNameData } = usePrimaryName();
 
-  useEffect(() => {
-    // Reset transaction state on unmount - clears transaction success banner
-    return () => {
-      dispatchTransactionState({ type: 'reset' });
-    };
-  }, []);
+  const [logoId, setLogoId] = useState<string | undefined>();
 
   useEffect(() => {
     if (!name) {
       navigate('/manage/names');
       return;
     }
+    // Reset transaction state on unmount - clears transaction success banner
```

Comment:
This change makes it where it resets on every name change rather than unmount. Double checking that is what you want; if so, change the comment.

Reply:
Yup thats fine, if the arns name match changes we want to reset the state AS WELL as on unmount